eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -199,9 +205,26 @@ private static XML program(final ZonedDateTime time, final String something) {
                     .add("program")
                     .attr("name", "main")
                     .attr("time", time.format(DateTimeFormatter.ISO_INSTANT))
+                    .add("objects")
                     .attr("something", something)
                     .up()
             ).xmlQuietly()
         );
     }
+
+    /**
+     * Adds attribute "hash" in EO program for tests.
+     * @param xml XML.
+     * @return XML representation of program.
+     */
+    private static XML updatedProgram(final XML xml)
+        throws NoSuchAlgorithmException, ImpossibleModificationException {
+        final String hash = new StHash.Hash(xml).getHash();
```

Comment:
@Yanich96 Is it possible to inline this variable?
https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html